src/picongpu/include/simulationControl/MySimulation.hpp

```diff
@@ -342,10 +342,10 @@ class MySimulation : public SimulationHelper<simDim>
         if( step != 0 )
         {
             namespace nvfct = PMacc::nvidia::functors;
-            (*pushBGField)( fieldE, nvfct::Sub(), fieldBackgroundE(fieldE->getUnit()),
-                            step, fieldBackgroundE::InfluenceParticlePusher);
-            (*pushBGField)( fieldB, nvfct::Sub(), fieldBackgroundB(fieldB->getUnit()),
-                            step, fieldBackgroundB::InfluenceParticlePusher);
+            (*pushBGField)( fieldE, nvfct::Sub(), FieldBackgroundE(fieldE->getUnit()),
```

Comment:
As discussed outside this review, this change is not so critical that we need to open an independent issue.